feat(documentation): adds container stories (#2030)
swisspost/design-system: packages/documentation/src/stories/foundation/layout/container/container.docs.mdx

import { Meta, Canvas } from '@storybook/blocks';
import * as ContainerStories from './container.stories';

<Meta of={ContainerStories} />

# Container

> Containers give resonsive paddign and margins to contents on a webpage. As our Grid is Based on Bootstrap negative Margins are used. Therefore Containers are required to wrap the Grid to prevent unintended side-effects.

Reviewer: ```suggestion
Containers give responsive padding and margins to contents on a webpage. As our grid is based on Bootstrap, negative margins are used. Therefore, containers are required to wrap the grid to prevent unintended side-effects.
```

<Canvas of={ContainerStories.Default} />

# Fluid

> `.container-fluid` has no padding regardless of viewport by default

Reviewer: ```suggestion
`.container-fluid` has no padding regardless of viewport by default.
```

There is the option to specify a breakpoint `.container-fluid-[breakpoint]` which will add a paddin for this breakpoint till the next bigger breakpoint.

# Reset

> `.container-reset` can be used on Elements inside a container so the padding effect can be prevented. This is done with a negative margin. Do not use this outside of a container or inside a fluid container.

Reviewer: ```suggestion
`.container-reset` can be used on elements inside a container, so the padding effect can be prevented. This is done with a negative margin. Do not use this outside of a container or inside a fluid container.
```
